src/webTypes/lib/vuelessConfig.js

```diff
@@ -0,0 +1,38 @@
+import fs from "node:fs";
+import path from "node:path";
+import esbuild from "esbuild";
+
+/**
+ * Load Vueless config from the project root.
+ * IIFE is used to prevent top level await issue.
+ */
+export let vuelessConfig = {};
```

Comment:
there is no reason to duplicated the code. Just pass vueless config as function argument in vite plugin.